app/components/wallet/WalletRestoreVerifyDialog.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../themes/mixins/loading-spinner';
+@import '../../themes/mixins/error-message';
+
+.dialog {
+  font-family: var(--font-light);
+  .walletPassword {
```

Comment:
I assume `.walletPassword` and `.error` shouldn't be here and are just copy pasted?

Reply:
Yep. My bad. Removed